README.md

```diff
@@ -1,2 +1,2 @@
-# jetpack
+# Jetpack
 A development repo for porting privly-firefox to Jetpack
```

Comment:
The readme should specify the command used to get started with testing the extension.